contrib/platform/test/com/sun/jna/platform/unix/X11Test.java

```diff
@@ -76,6 +76,12 @@ public void testXFetchName() {
         }
     }
 
+    public void testSetWMProtocols() {
+        X11.Atom[] protocols = new X11.Atom[]{ X11.INSTANCE.XInternAtom(display, "WM_DELETE_WINDOW", false) };
+        int status = X11.INSTANCE.XSetWMProtocols(display, root, protocols, 1);
+        assertEquals("Bad status for XSetWMProtocols", 1, status);
```

Comment:
I read http://linux.die.net/man/3/xsetwmprotocols:

> If it cannot intern the WM_PROTOCOLS atom, XSetWMProtocols returns a zero status. Otherwise, it returns a nonzero status.

So - would it be better to check for a non-zero status?